schemas/README.md

```diff
@@ -0,0 +1,7 @@
+# README
+
+This directory contains JSON Schemas:
+
+* [`generic-serialization.schema.json`](./generic-serialization.schema.json): schema that captures the LIonWeb serialization JSON format _generically_.
+* [`lioncore.serialization.schema.json`](./lioncore.serialization.schema.json): schema that captures the LIonWeb serialization JSON format _specifically_ for serialized metamodels (i.e., instances of LIonCore/M3).
```

Comment:
Do we want to have both or is the generic one enough? Was this a point that we discussed?

Reply:
I don't think we discussed it. It serves first and foremost as a check that serialization also behaves correctly in view of a specific metamodel. The generic JSON Schema is rather...permissive, so I think there's value in a specific JSON Schema. E.g., someone could generate type definitions from a metamodel-specific JSON Schema.